.github/scripts/update_disabled_issues.py

```diff
@@ -26,92 +26,99 @@
+    _data = json.dumps(data).encode() if data is not None else None
+    try:
+        with urlopen(Request(url, headers=headers, data=_data)) as conn:
+            return json.load(conn)
+    except Exception as err:
+        print(f"Failed to get {url}: {err}")
 
 
-def get_last_page(header: Any) -> int:
-    # Link info looks like:
-    # Link: <https://api.github.com/search/issues?q=is%3Aissue+is%3Aopen+repo%3Apytorch%2Fpytorch+in%3Atitle+
-    # DISABLED&per_page=30&page=2>; rel="next", <https://api.github.com/search/issues?q=is%3Aissue+is%3Aopen+
-    # repo%3Apytorch%2Fpytorch+in%3Atitle+DISABLED&per_page=30&page=4>; rel="last"
-    link_info = header["link"]
-    if link_info is None:
-        print(
-            "WARNING: Link information missing, most likely because there is only one page of results."
-        )
-        return 1
-    prefix = "&page="
-    suffix = ">;"
-    return int(
-        link_info[link_info.rindex(prefix) + len(prefix) : link_info.rindex(suffix)]
+def gh_graphql(query: str, token: str, **kwargs: Any) -> Dict[str, Any]:
+    rc = github_api_request(
+        "https://api.github.com/graphql",
+        data={"query": query, "variables": kwargs},
+        token=token,
     )
-
-
-def update_issues(issues_json: Dict[Any, Any], info: str) -> None:
-    more_issues = json.loads(info)
-    issues_json["items"].extend(more_issues["items"])
-    issues_json["incomplete_results"] |= more_issues["incomplete_results"]
+    if "errors" in rc:
+        raise RuntimeError(
+            f"GraphQL query {query}, args {kwargs} failed: {rc['errors']}"
+        )
+    return cast(Dict[str, Any], rc)
 
 
 @lru_cache()
-def get_disable_issues(prefix: str = DISABLED_PREFIX) -> Dict[Any, Any]:
-    prefix = (
-        f"https://api.github.com/search/issues?q=is%3Aissue+is%3Aopen+repo:{OWNER}/{REPO}+in%3Atitle+{prefix}&"
-        "&per_page=100"
-    )
-    header, info = github_api_request(prefix + "&page=1")
-    issues_json = json.loads(info)
-    last_page = get_last_page(header)
-    assert (
-        last_page > 0
-    ), "Error reading header info to determine total number of pages of labels"
-    for page_number in range(2, last_page + 1):  # skip page 1
-        _, info = github_api_request(prefix + f"&page={page_number}")
-        update_issues(issues_json, info)
-
-    return issues_json
-
-
-def validate_and_sort(issues_json: Dict[str, Any]) -> None:
-    assert issues_json["total_count"] == len(issues_json["items"]), (
-        f"The number of issues does not equal the total count. Received {len(issues_json['items'])}, "
-        f"while the total count is {issues_json['total_count']}."
-    )
-    assert not issues_json[
-        "incomplete_results"
-    ], "Results were incomplete. There may be missing issues."
-
-    # score changes every request, so we strip it out to avoid creating a commit every time we query.
-    for issue in issues_json["items"]:
-        if "score" in issue:
```

Comment:
I couldn't remember what this score is about, but anyway I see the list of issues is still sorted in the same way.